README.md

```diff
@@ -9,16 +9,4 @@ Requirements:
 
 ## License
 
-Licensed under either of
-
- * Apache License, Version 2.0, ([LICENSE-APACHE](LICENSE-APACHE) or http://www.apache.org/licenses/LICENSE-2.0)
- * MIT license ([LICENSE-MIT](LICENSE-MIT) or http://opensource.org/licenses/MIT)
-
-at your option.
-
-### Contribution
-
-Unless you explicitly state otherwise, any contribution intentionally
-submitted for inclusion in the work by you, as defined in the Apache-2.0
-license, shall be dual licensed as above, without any additional terms or
-conditions.
+This repository is [MIT/Apache 2.0 dual-licensed](COPYING).
```

Comment:
The existing preamble above is an ecosystem-wide standard for Rust crates, because what crates.io shows is the `README`, and the `Cargo.toml`'s `license` field, not any `COPYING` file. I would greatly prefer that the above preamble is left exactly as-is, and we instead duplicate it in `COPYING` for GitHub's benefit.

Reply:
(I say this here because @daira suggested elsewhere duplicating this pattern; I'm ambivalent about this pattern here in Python code.)

Reply:
I will revert this and update the other PRs.

Reply:
Addressed in #60.